libbeat/docs/security/users.asciidoc

```diff
@@ -29,11 +29,11 @@ strategy.
 
 IMPORTANT: Setting up {beatname_uc} is an admin-level task that requires extra
 privileges. As a best practice, grant the setup role to administrators only, and
-use a less restrictive role for event publishing.  
+use a less restrictive role for event publishing.
```

Comment:
Shouldn't this say `more restrictive` or `less privileged`?

Reply:
it definitely should